moto/ssoadmin/models.py

```diff
@@ -83,13 +96,27 @@ def generate_id(instance_arn: str) -> str:
         )
 
 
+class ManagedPolicy(BaseModel):
+    def __init__(self, arn: str, name: str):
+        self.arn = arn
+        self.name = name
+
+    def __eq__(self, other: Any) -> bool:
+        if not isinstance(other, ManagedPolicy):
+            return False
+        return self.arn == other.arn
+
+
 class SSOAdminBackend(BaseBackend):
     """Implementation of SSOAdmin APIs."""
 
     def __init__(self, region_name: str, account_id: str):
         super().__init__(region_name, account_id)
         self.account_assignments: List[AccountAssignment] = list()
         self.permission_sets: List[PermissionSet] = list()
+        self.aws_managed_policies: Dict[str, Any] = json.loads(
```

Comment:
This is quite a big JSON file, so loading is relatively expensive. Can we lazy-load this, so that it's only called when used in `attach_managed_policy_to_permission_set`?

That way, people who do not use this method (and don't need the data) aren't impacted.

Reply:
Good call!  Fixed this.  Let me know if this change resolves your concern @bblommers .